deployment/terraform/task-definitions/chatta.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "name": "gt-chatta",
+        "image": "${chatta_image}",
+        "cpu": 10,
+        "memory": 2048,
+        "essential": true,
+        "portMappings": [
+            {
+                "containerPort": 8777,
+                "hostPort": 0
+            }
```

Comment:
Looks like I missed this in the Transit review, but I think that we should route logs to CloudWatch Logs for these services.